setup.py

```diff
@@ -47,5 +48,9 @@
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Framework :: Django',
+        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.0',
```

Comment:
after this change, 2.1 is also supported, right?
```suggestion
        'Framework :: Django :: 2.0',
        'Framework :: Django :: 2.1',
```

Reply:
isn't the 2.1 marker missing still?